contracts/KlerosPOC.sol

```diff
@@ -691,6 +692,22 @@ contract KlerosPOC is Arbitrator {
         return dispute.voteCounter[dispute.appeals].winningChoice;
     }
 
+    /** @dev Return the repartitioned token amount for a juror in a specific appeal.
+     *  @param _disputeID ID of the dispute.
+     *  @param _appeal The appeal of the dispute.
+     *  @param _ruling The ruling of the juror.
+     *  @return repartitionedPNK The amount of PNK lost or won through the repartition.
+     */
+    function repartitionedPNK(uint _disputeID, uint _appeal, uint _ruling) public constant returns(uint repartitionedPNK) {
+        Dispute storage dispute = disputes[_disputeID];
+        return _ruling == currentRuling(_disputeID) ?
```

Comment:
Ok, now I see why you put everything in storage. For this function.
Do we really need this function? I mean does another smart contract need to read it?
Otherwise if it’s just for display events would be better.